### PR TITLE
Component/button bar

### PR DESCRIPTION
**Pull request by davodey**
A responsive button tool bar that responds to the recommended gravity breakpoints.

### CODE REVIEW

**Comment by jackiejakob** [comment]
This looks great! Here are a few notes from me:
1. The svgs are set to fill: #252525 and aren't inheriting the button color of #6a7070 (assuming we want icon buttons to be dark grey). Also thinking more about this, maybe we should offer the color mode like you do for the header? Or would you think users can customize the buttons since those are in the slot anyway?

2. Can you override the browser default ul margin to zero bc it's coming in as 86px tall when I added it into my header component?

3. I think we can make the "more" button the same size as the others for equal spacing?
image

4. I noticed that if you align the button bar to the right of the screen, the dropdown pushes against the edge (and shifts the buttons over): 
image
Are there plans to have the dropdown adjust alignment based on position in the page?

5. VO isn't reading the button labels for me but I'll let EJ comment more on a11y.

6. Is the primary use for this bar to go inside of the gr-header?  I run into styling issues when I did that. E.g., the button padding goes to zero for buttons inside gr-header. 

Wasn't sure how you're setting things up now to help cross some of these off. I see a gravity-css/button-bar stylesheet now instead of css included in the component.

**Comment by davodey** [comment]
@jackiejakob issues 1-3 have been fixed and ready for re-review.

4. I will look into the vo issue and see how to fix.
5. This bar is not meant to go inside the gr-header, its meant to go in a page frame that contains the alternate header with a title and X to the left.

**Comment by davodey** [comment]
VO issue should be fixed.

**Comment by jackiejakob** [comment]
@davodey when I put the button bar inside my header, the dropdown menu doesn't open to the left:
image
Should I be adding anything to make that happen?

The aria-expanded on the dropdown icon doesn't turn to true when I open the menu. And maybe @dengeist can speak to the tab behavior when he's back -- should the menu close if you shift + tab out of it or tab forward out of it?

File: button-bar/index.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          <svg data-text="Edit" focusable="false" class="icon-18" aria-hidden="true">
+            <path d="M9.26201579,1.31563727 C10.3665853,1.31563727 11.2620158,2.21106777 11.2620158,3.31563727 L11.2620158,14.8529591 L9.132426,17.0114903 C8.93848613,17.2080658 8.62191085,17.2102024 8.42533532,17.0162626 L6.26201579,14.8529591 L6.26201579,3.31563727 C6.26201579,2.21106777 7.15744629,1.31563727 8.26201579,1.31563727 L9.26201579,1.31563727 Z M8.01201579,6.31563727 L8.01201579,14.1280854 L8.77050889,14.8865785 L9.51201579,14.1349944 L9.51201579,6.31563727 L8.01201579,6.31563727 Z M8.01201579,4.64897061 L9.51201579,4.64897061 L9.51201579,3.31563727 C9.51201579,3.17756609 9.40008698,3.06563727 9.26201579,3.06563727 L8.26201579,3.06563727 C8.1239446,3.06563727 8.01201579,3.17756609 8.01201579,3.31563727 L8.01201579,4.64897061 Z" fill-rule="nonzero" transform="translate(8.762016, 9.237984) rotate(45.000000) translate(-8.762016, -9.237984) "></path>
+          </svg>
+        </button>
+      </li>
+      <li data-text="Bookmark" slot="list">
+        <button class="gr-btn icon-btn-18 ">
+          <svg data-text="Edit" focusable="false" class="icon-18" aria-hidden="true">
```
Comment (jackiejakob) [inline button-bar/index.html]: I assume the data-text should change on the SVGs too?

**Pull request merged**